examples/confidential-coins/scripts/test_confidential_coins

```diff
@@ -51,7 +51,7 @@ pushd _test_confidential_coins
     AGGREGATOR_KEYFILE="aggregator.key"
     SNARKJSVKFILE="../circuits/snarkjs_verification_key.json"
     APP_VKFILE="../circuits/upa_verification_key.json"
-    UPAINSTANCE="upa.instance"
+    UPAINSTANCE="upa.instance.json"
```

Comment:
Should we go the whole way and do `upa-instance.json`?  Or maybe `upa_instance.json`.
We probably have some inconsistencies re `-` vs `_`, but introducing `upa.instance.json` might look like yet another convention.